src/cli/run.rs

```diff
@@ -15,6 +15,10 @@ use container::Container;
 /// The `handler` method provided below will be executed.
 #[derive(Debug, Args)]
 pub struct RunCommand {
+    /// Name of the container instance that will be start. It must me unique on your host
+    #[clap(name = "container-id")]
```

Comment:
If it's the container name, then let's just use `--name`.

Reply:
As the name of the container must be unique to the host I was thinking that `id` was best to describe this argument.